obm-autoset/README.md

```diff
@@ -0,0 +1,46 @@
+#RackHD OBM Setting Auto Set Tool
+Tool to apply OBM setting for each compute node discovered by RackHD ora. Save the trouble to manually apply OBM setting node by node.
+
+##Installation and execution
+clone this repository.
+cd to the cloned directory.
+change the environment setting in obm_setting.json file.
+python ApplyOBMSetting.py -ora 192.168.129.96
+
+##Theroy of Operation
+The script will check the RMM/BMC address infomation in the catalog of ora. And then put OBM setting to ora by RackHD 2.0 API. The compute node should be discovered by RackHD ora first.
+
+##Supported parameters
+###-ora
+Specify RackHD ora IP address, default use localhost
+
+###-user
+Use User specifed BMC Name instead of the usernames in OBM setting file, default is "None"
+
+###-password
+Use User specifed BMC password instead of the password in OBM setting file, default is "None"
+
+
+
+###-v {0,2,4,6,9}
+Verbosity level of console output:

```

Comment:
This is just a simple script, I don't see a necessary to introduce so many log levels.